.github/scripts/release_code.sh

```diff
@@ -2,16 +2,65 @@
 
 echo "$COMMIT_ID"
 
+# Fetch the artifact bucket
 artifact_bucket=$(aws cloudformation list-exports --output json | jq -r '.Exports[] | select(.Name == "account-resources:ArtifactsBucket") | .Value' | grep -o '[^:]*$')
 export artifact_bucket
 
+# Fetch the CloudFormation execution role
 cloud_formation_execution_role=$(aws cloudformation list-exports --output json | jq -r '.Exports[] | select(.Name == "ci-resources:CloudFormationExecutionRole") | .Value' )
 export cloud_formation_execution_role
 
-# TRUSTSTORE_BUCKET_ARN=$(aws cloudformation describe-stacks --stack-name account-resources --query "Stacks[0].Outputs[?OutputKey=='TrustStoreBucket'].OutputValue" --output text)
-# TRUSTSTORE_BUCKET_NAME=$(echo "${TRUSTSTORE_BUCKET_ARN}" | cut -d ":" -f 6)
-# LATEST_TRUSTSTORE_VERSION=$(aws s3api list-object-versions --bucket "${TRUSTSTORE_BUCKET_NAME}" --prefix "${TRUSTSTORE_FILE}" --query 'Versions[?IsLatest].[VersionId]' --output text)
-# export LATEST_TRUSTSTORE_VERSION
+# Fetch trust store details
+TRUSTSTORE_BUCKET_ARN=$(aws cloudformation describe-stacks --stack-name account-resources --query "Stacks[0].Outputs[?OutputKey=='TrustStoreBucket'].OutputValue" --output text)
+TRUSTSTORE_BUCKET_NAME=$(echo "${TRUSTSTORE_BUCKET_ARN}" | cut -d ":" -f 6)
+LATEST_TRUSTSTORE_VERSION=$(aws s3api list-object-versions --bucket "${TRUSTSTORE_BUCKET_NAME}" --prefix "${TRUSTSTORE_FILE}" --query 'Versions[?IsLatest].[VersionId]' --output text)
+export LATEST_TRUSTSTORE_VERSION
 
+if [ -z "${DOMAIN_NAME_EXPORT}" ]; then
+  export DOMAIN_NAME_EXPORT="NOT_SET"
+fi
+
+if [ -z "${ZONE_ID_EXPORT}" ]; then
+  export ZONE_ID_EXPORT="NOT_SET"
+fi
+
+# Fetch the VPC and subnets
+VPC=$(aws cloudformation list-exports --output json | jq -r '.Exports[] | select(.Name == "DefaultVPCId") | .Value')
```

Comment:
I dont see the need for any of this onwards, you dont need to set these values as env vars, they are all exported from account_resources and can be freely imported directly in the templates. In addition they are using the wrong set of subnets, the LB should be in the public subnets.

Reply:
Redundant values have been removed.